src/test/groovy/graphql/schema/idl/SchemaGeneratorTest.groovy

```diff
@@ -542,13 +542,6 @@ class SchemaGeneratorTest extends Specification {
             extend type BaseType {
                extraField5 : Boolean!
             }
-            #
-            # if we repeat a definition, that's ok as long as its the same types as before
-            # they will be de-duped since the effect is the same
-            #
-            extend type BaseType implements Interface1 {
-               extraField1 : String
-            }
```

Comment:
Why does this need to be removed?  Is it no longer valid or just not longer relevant and tested in the other places

Reply:
This is actually invalid now. 
I've added a new validation rule which prevents extensions from implementing an interface which has already been implemented from a base type, or other extensions for the same type.
This will make the graphql-java validation more aligned with graphql-js